ozaria/site/components/teacher-dashboard/common/SecondaryTeacherNavigation.vue

```diff
@@ -60,13 +60,23 @@ export default {
       // TODO: do show the assessments if it is CodeNinjas, but not in a camp context
       return utils.isCodeCombat && !me.isCodeNinja()
     },
+
+    sortedClasses () {
+      const classrooms = [...this.classrooms]
+      classrooms.sort(this.classroomSortById)
+      return classrooms
+    },
   },
 
   methods: {
     isCurrentRoute (route) {
       return this.$route.path.startsWith(route)
     },
 
+    classroomSortById (a, b) {
```

Comment:
a good idea to reuse the existing function by moving it to a util or creating a vue mixin that can be re-used